rmm/armv9a/src/realm/registry.rs

```diff
@@ -147,7 +145,8 @@ impl monitor::rmi::Interface for RMI {
         prot: usize,
     ) -> Result<(), &str> {
         let mut flags = 0;
-        let mut realm_pas = true;
+        let _prot = MapProt::new(prot);
+        let mut realm_pas: bool = _prot.is_set(MapProt::NS_PAS) == false;
```

Comment:
`_`는 사용하지 않는 variable에 붙이니 제거하는 것이 좋을 것 같습니다.


```suggestion
        let prot = MapProt::new(prot);
        let mut realm_pas = !prot.is_set(MapProt::NS_PAS);
```